ckanext/s3filestore/commands.py

```diff
@@ -45,8 +45,9 @@ def check_config(self):
         S3_conn = boto.connect_s3(public_key, secret_key)
 
         # Check if bucket exists
-        bucket = S3_conn.lookup(bucket_name)
-        if bucket is None:
+        try:
+            bucket = S3_conn.get_bucket(bucket_name)
+        except boto.exception.S3ResponseError as err:
             print 'Bucket {0} does not exist, trying to create it...'.format(
```

Comment:
```suggestion
            print 'Bucket {0} does not exist. You should create it or ensure access permissions are correct.'.format(
```

Reply:
erm, it does attempt to create the bucket below

Reply:
It does? It never has permission though does it?

Reply:
in our deployments no, but we'd like to keep it as close as possible from the original repo so that was my mind set. It is worth considering though, attempting to create a bucket is ridiculous.